src/legacy/core_plugins/kibana/migrations.js

```diff
@@ -264,7 +264,7 @@ export const migrations = {
       }
     },
     '7.0.1': removeDateHistogramTimeZones,
-    '7.1.0': doc => executeMigrations710(doc)
+    '7.2.0': doc => executeMigrations720(doc)
```

Comment:
nit: As this function is unary, you could remove the extra function and pass this by reference:

```js
'7.2.0': executeMigrations720,
```